public/index.html

```diff
@@ -28,17 +33,17 @@
   </head>
   <body>
 
-  <!-- Import jQuery before materialize.js -->
-  <script src="https://code.jquery.com/jquery-2.1.1.min.js"></script>
-  <script src="https://cdnjs.cloudflare.com/ajax/libs/materialize/0.98.0/js/materialize.min.js"></script>
-  <!-- And then your bundled js -->
-
 
-  <noscript>
-    You need to enable javascript to run this app
-  </noscript>
- <div id="root"></div>
 
+  <!-- Import jQuery before materialize.js -->
+   <script src="https://code.jquery.com/jquery-2.1.1.min.js"></script>
+   <script src="https://cdnjs.cloudflare.com/ajax/libs/materialize/0.98.0/js/materialize.min.js"></script>
+  <!-- And then your bundled js -->
+  <script src="path/to/your/bundle.js"></script>
```

Comment:
I don't think we need this line too

Reply:
@isheebo , I have implemented this feedback.